CONFIGURATION.md

```diff
@@ -262,6 +262,11 @@ validate_additional_rrs:
 # The size of the payload.
 [ payload_size: <int> ]
 
+# TTL of outbound packets. Can be used to test reachability of a target
+# within a given number of hops, for example, to determine when network
+# routing has changed.
```

Comment:
Please add a note about the valid range being [0, 255], and that 0 means use the default value of 64... but that bit doesn't seem to be true. It looks like there's a path where the default is 64 and another one where it defaults to the current setting in the OS.

Reply:
I've included this in commit 6c29e82.

See next comment about discussion of using the OS's default versus a default baked into blackbox_exporter.